### PR TITLE
Fix use of % operator on string with multiple arguments

### PR DESCRIPTION
**Pull request by bjodah**
This is a minor fix, but the incorrect format strings raised a `TypeError`:
```
TypeError: not enough arguments for format string
```
Don't know if we want this in for 1.1?

cc @asmeurer @siefkenj

### CODE REVIEW

File: sympy/matrices/matrices.py [hunk 1]
```diff
@@ -4010,12 +4010,12 @@ def a2idx(j, n=None):
         try:
             j = j.__index__()
         except AttributeError:
-            raise IndexError("Invalid index a[%r]" % (j,))
+            raise IndexError("Invalid index a[%r]" % j)
```
Comment (asmeurer) [inline sympy/matrices/matrices.py]: This was actually correct. The other way does the wrong thing if `j` is a tuple.

**Comment by smichr** [comment]
+1

File: sympy/matrices/matrices.py [hunk 2]
```diff
@@ -4015,7 +4015,7 @@ def a2idx(j, n=None):
         if j < 0:
             j += n
         if not (j >= 0 and j < n):
-            raise IndexError("Index out of range: a[%s]" % (j,))
+            raise IndexError("Index out of range: a[%s]" % j)
```
Comment (asmeurer) [inline sympy/matrices/matrices.py]: And also this one.

Comment (bjodah) [inline sympy/matrices/matrices.py]: Really? I think `j` is guaranteed to be an integral type at this point?

File: sympy/matrices/matrices.py (hunk 1 above)
Comment (smichr) [inline sympy/matrices/matrices.py]: Yes, here.

File: sympy/matrices/matrices.py (hunk 2 above)
Comment (smichr) [inline sympy/matrices/matrices.py]: The mod is right here.

File: sympy/matrices/matrices.py (hunk 1 above)
Comment (smichr) [inline sympy/matrices/matrices.py]: I'm actually a little confused. I thought this function was in common. My guess is that the common functions have been moved to matrices. But whatever the case, I believe the first case was previously correct and the second case is correct after modifications.

File: sympy/matrices/matrices.py (hunk 2 above)
Comment (asmeurer) [inline sympy/matrices/matrices.py]: Oh you are right.

**Pull request merged**